app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controller/controller')
+
+app.use(express.json());
```

Comment:
we're not using this line yet, please remove it until the code requires you to add it in